content/sensu-go/6.3/observability-pipeline/observe-schedule/business-service-monitoring-sdk.md

```diff
@@ -0,0 +1,125 @@
+weight: 75
+version: "6.3"
+product: "Sensu Go"
+platformContent: false 
+menu:
+  sensu-go-6.3:
+    parent: observe-schedule
+---
+
+**COMMERCIAL FEATURE**: Access business service monitoring (BSM) in the packaged Sensu Go distribution.
+For more information, see [Get started with commercial features][4].
+
+Sensu's business service monitoring (BSM) feature uses a dedicated SDK of JavaScript-based expressions that provide additional functionality.
+Use the BSM SDK to create custom JavaScript expressions with complex logic.
+
+BSM SDK expressions are defined in [rule templates][3], so they act in the context of determining whether aggregate data derived from a service component’s selection of Sensu Go events should trigger a rule-based event.
+They always receive a single event and some information about that event, like `event.timestamp` or `event.check.interval`, and always return either `true` or `false`.
+
+BSM SDK expressions are evaluated by the [Otto JavaScript VM][1] as JavaScript programs.
+
+## Syntax quick reference
+
+<table>
+<thead>
+<tr>
+<th>operator</th>
+<th>description</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td><code>===</code></td>
+<td>Identity</td>
+</tr>
+<tr>
+<td><code>!==</code></td>
+<td>Nonidentity</td>
+</tr>
+<tr>
+<td><code>==</code></td>
+<td>Equality</td>
+</tr>
+<tr>
+<td><code>!=</code></td>
+<td>Inequality</td>
+</tr>
+<tr>
+<td><code>&&</code></td>
+<td>Logical AND</td>
+</tr>
+<tr>
+<td><code>||</code></td>
+<td>Logical OR</td>
+</tr>
+<tr>
+<td><code><</code></td>
+<td>Less than</td>
+</tr>
+<tr>
+<td><code>></code></td>
+<td>Greater than</td>
+</tr>
+<tr>
+<td><code><=</code></td>
+<td>Less than or equal to</td>
+</tr>
+<tr>
+<td><code>>=</code></td>
+<td>Greater than or equal to</td>
+</tr>
+</tbody>
+</table>
+
+## Specification
+
+BSM SDK expressions are valid ECMAScript 5 (JavaScript) expressions that return either `true` or `false`.
+Other values are not allowed.
+If an expression returns a value besides `true` or `false`, the [Sensu backend log][2] will record an error and the filter will evaluate to `false`.
+
+The BSM SDK allows you to to express rules for the number or percentage of events with critical, warning, OK, and unknown statuses.
+You can also configure expressions to ignore silenced events.
+
+## Custom functions
+
+The Sensu BSM SDK includes two custom functions: `sensu.Count()` and `sensu.Percentage()`.
+
+### sensu.Count()
+
+The custom function `sensu.Count()` returns the number of events with the specified status.
+For example, to return the number of events with `ok` status:
+
+{{< code go >}}
+sensu.Count("ok")
+{{< /code >}}
+
+### sensu.Percentage()
+
+The custom function `sensu.Percentage()` returns the percentage of events with the specified status.
+For example, to return the percentage of events with `critical` status:
+
+{{< code go >}}
+sensu.Percentage("critical")
+{{< /code >}}
+
+## Example BSM SDK expression
+
```

Comment:
I added this example based on notes I found from [Simon's spike](https://groups.google.com/a/sensu.io/g/eng-discuss/c/ok8rexkqi0E/m/YbUjVAjwBQAJ?pli=1)